assets/constants.ts

```diff
@@ -115,6 +115,22 @@ export default {
     "IncreasingUpdatedOn": "Most Recently Updated",
     "DecreasingUpdatedOn": "Least Recently Updated"
   },
+
+  // Sorting categories for contributor admin dashboard.
+  "CD_ADMIN_STATS_SORT_OPTIONS": {
```

Comment:
You should have a backend test that verifies that the keys here match the constants defined in the backend file (see comment below). This will help catch errors if a developer changes the key in the future.

Reply:
Made the sort_choices more strict below PTAL.

Reply:
Could you please point me to the backend test? There should still be one to ensure that the constants defined here and the ones defined in the backend don't diverge.

Reply:
Where should this backend test go?

Reply:
I would put it in the test file associated with the backend file which defines those constants. In this case they would go in core/storage/suggestions/gae_models_test.py.

Reply:
Added it there. PTAL.